packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -260,144 +311,205 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
       return
     }
 
+    // TODO: Should we save this network info in LevelDB at this point?
     return network
   }
 
-  private async generatePSK() {
-    const pskBase64 = Libp2pService.generateLibp2pPSK().psk
-    await this.localDbService.put(LocalDBKeys.PSK, pskBase64)
-    this.logger('Generated Libp2p PSK')
-    this.serverIoProvider.io.emit(SocketActionTypes.LIBP2P_PSK_STORED, { psk: pskBase64 })
-  }
```

Comment:
We store the psk in the Community object, so there is no more need for the PSK key in LevelDB. We also return the entire Community object from createCommunity so there is no more need for the LIBP2P_PSK_STORED event.